core/CronArchive.php

```diff
@@ -429,13 +429,12 @@ public function runScheduledTasks()
 
         $this->log("Starting Scheduled tasks... ");
 
-        $tasksOutput = $this->request("?module=API&method=CoreAdminHome.runScheduledTasks&format=csv&convertToUnicode=0&token_auth=" . $this->token_auth);
+        API\Request::processRequest("CoreAdminHome.runScheduledTasks", array(
+            'token_auth' => $this->token_auth,
+            'format' => 'original', // so exceptions get thrown
+            'trigger' => 'archivephp'
+        ));
 
-        if ($tasksOutput == \Piwik\DataTable\Renderer\Csv::NO_DATA_AVAILABLE) {
```

Comment:
Can you here check maybe for the response of the `runScheduledTasks` method and log this message if `$exectionResults` is an empty array? But I think I saw you added some logger statements in the Scheduler as well so maybe it is not needed here anymore :) Ignore it in this case :)